src/main/java/homework_2/traffic_light/TrafficLight.java

```diff
@@ -0,0 +1,83 @@
+package homework_2.traffic_light;
+
+import homework_2.traffic_light.exception.TrafficLightException;
+import homework_2.traffic_light.utils.MessageType;
+
+import static homework_2.traffic_light.utils.MessageType.INFO_MESSAGE;
+import static homework_2.traffic_light.utils.MessageType.INFO_MESSAGE_TO_MODE_0;
+import static homework_2.traffic_light.utils.MessageType.INFO_MESSAGE_TO_MODE_1;
+import static homework_2.traffic_light.utils.MessageType.LIGHT_GREEN_MESSAGE;
+import static homework_2.traffic_light.utils.MessageType.LIGHT_RED_MESSAGE;
+import static homework_2.traffic_light.utils.MessageType.LIGHT_YELLOW_MESSAGE;
+import static homework_2.traffic_light.utils.MessageType.ERROR_MESSAGE_EXCEED_LIMIT;
+import static homework_2.traffic_light.utils.MessageType.ERROR_MESSAGE_INCORRECT_FORMAT;
+import static homework_2.traffic_light.utils.MessageType.ERROR_MESSAGE_NEGATIVE_NUMBERS;
+import static homework_2.traffic_light.utils.Utils.getData;
+import static homework_2.traffic_light.utils.Utils.printMessage;
+
+public class TrafficLight {
```

Comment:
55 gives me RED, but should YELLOW
![image](https://user-images.githubusercontent.com/58872509/126618959-714c4c0b-bd9c-42f0-ad52-3b305c83484f.png)

Reply:
I hadn't checked with updates in a homework assignment, so that's why I made this mistake. It should be ok now